### PR TITLE
pervasive: adding method Map::contains_value

### PR DESCRIPTION
**Pull request by achreto**
This is the dual of Map::contains_key but it checks whether the map contains a specific value.

### CODE REVIEW

File: source/pervasive/map.rs [hunk 1]
```diff
@@ -107,6 +107,13 @@ impl<K, V> Map<K, V> {
         self.dom().contains(k)
     }
 
+    /// Returns true if the value `v` is in the map of `self`.
+
+    #[verifier(inline)]
+    pub open spec fn contains_value(self, v: V) -> bool {
+        exists|i: K| #[trigger] self.dom().contains(i) && self.index(i) == val
```
Comment (utaal) [inline source/pervasive/map.rs:114]: Sorry I didn't spot this earlier, can you use `self[i]` instead?

Comment (achreto) [reply source/pervasive/map.rs]: Done!

**Comment by utaal** [comment]
I added a test, which ends up generating invalid AIR. I don't know what the issue is yet.

**Comment by utaal** [comment]
It was a problem with `#[verifier(inline)]` which interacts badly with native types as type params here.

**Pull request merged**